src/test/java/test_cases/BaseTest.java

```diff
@@ -0,0 +1,20 @@
+package test_cases;
+
+import common.helpers.BrowserHelper;
+import common.helpers.Constant;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+
+public class BaseTest {
+
+  @BeforeMethod
+  public void beforeMethod() {
+    BrowserHelper.startBrowser(BrowserHelper.DriverType.CHROME);
```

Comment:
hardcode Chrome here. How do you use Firefox or Edge?

Reply:
> hardcode Chrome here. How do you use Firefox or Edge?

if using FireFox or Edge, I change CHROME to FIREFOX or EDGE. All 3 I have declared the in the BrowserHelper class and  in the startBrowser method

- FireFox 
`BrowserHelper.startBrowser(BrowserHelper.DriverType.FIREFOX);`

-Edge
`BrowserHelper.startBrowser(BrowserHelper.DriverType.EDGE);`